### PR TITLE
OFFAPPS-538 Updated translations

### PR DESCRIPTION
**Pull request by ocke**
:koala:

Describe your PR

/cc @zendesk/quokka

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-538

### Risks
 - None

### CODE REVIEW

File: translations/cs.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "app": {
     "parameters": {
       "iframeURL": {
-        "label": "URL prvku iFrame"
+        "label": "URL elementu iFrame"
```
Comment (sandlerr) [inline translations/cs.json]: iframe is lowercase in every other language

Comment (xuebinglee) [inline translations/cs.json]: Wow! Good catch!

Comment (ocke) [inline translations/cs.json]: Sent complains to the translation team...

**Comment by ocke** [comment]
@ydrain

**Pull request merged**